src/styles/components/buttonStyles.ts

```diff
@@ -18,6 +18,17 @@ const buttonStyles = {
       _hover: { bg: "brand.secondary", opacity: "0.60" },
       _active: { bg: "brand.secondary", opacity: "0.60" },
     },
+    checkoutBtn: {
+      bg: "brand.secondary",
+      color: "white",
+      size: "lg",
+      fontSize: "lg",
+      fontWeights: "bold",
+      lineHeights: "22",
+      borderRadius: "6px",
+      _hover: { bg: "brand.secondary", opacity: "0.60" },
+      _active: { bg: "brand.secondary", opacity: "0.60" },
+    },
```

Comment:
Seems the same as the shareBtn style on the same file. Duplicated code.

Reply:
I think it is different. The background colour and size is very strange If I use shareBtn style for drop-down button.